molgenis-gavin/src/main/java/org/molgenis/gavin/exception/JobNotFoundException.java

```diff
@@ -0,0 +1,25 @@
+package org.molgenis.gavin.exception;
+
+import org.molgenis.i18n.CodedRuntimeException;
+
+public class JobNotFoundException extends CodedRuntimeException
```

Comment:
remove and replace occurences with UnknownEntityException

Reply:
assign